src/test/java/uk/gov/hmcts/reform/sendletter/services/date/holidays/BankHolidaysClientTest.java

```diff
@@ -19,16 +19,21 @@
 
 public class BankHolidaysClientTest {
 
-    @Rule
-    public WireMockRule api = new WireMockRule();
-
     private BankHolidaysClient client;
+    private WireMockServer api;
 
-    @Before
+    @BeforeEach
     public void setUp() {
+        this.api = new WireMockServer();
```

Comment:
looks like can be static final, does it not? then start/stop per class instance rather than test case

Reply:
the recommended
```
@Rule
public WireMockRule wireMockRule = new WireMockRule();
```
starts and stops the server for each test, sticking to that.

http://wiremock.org/docs/junit-rule/